### PR TITLE
A simple example that illustrates selective drawing of BufferGeometry

### PR DESCRIPTION
**Pull request by callumprentice**
As per our discussion in Twitter - a simple example that illustrates selective drawing/culling of BufferGeometry via a shader and the `discard` statement.

Initial implementation suggestion via WestLangley on Stack Overflow.

### CODE REVIEW

File: examples/webgl_buffergeometry_selective_draw.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+		}
+	</script>
+	</head>
+	<body onload="app()">
+		<div id="title"></div>
+		<div id="ui"><a href="#" onclick="hide_lines();">CULL SOME LINES</a> - <a href="#" onclick="show_all_lines();">SHOW ALL LINES</a></div>
+		<script>
+		var camera, scene, renderer, stats;
+		var geometry, mesh;
+		var num_lat = 100;
```
Comment (liammagee) [inline examples/webgl_buffergeometry_selective_draw.html]: Maybe use camel case (numLat or just lat) here and for function names below.

File: examples/webgl_buffergeometry_selective_draw.html [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+	<script type="text/javascript" src="js/libs/stats.min.js"></script>
+
+	<script type="x-shader/x-vertex" id="vertexshader">
+		attribute float visible;
+		varying float vVisible;
+		attribute vec3 vertColor;
+		varying vec3 vColor;
+
+		void main() {
+			gl_Position = projectionMatrix * modelViewMatrix * vec4(position,1.0);
```
Comment (liammagee) [inline examples/webgl_buffergeometry_selective_draw.html]: Should be last line in the function?

**Comment by liammagee** [comment]
This looks like a useful example, thanks.

File: examples/webgl_buffergeometry_selective_draw.html (hunk 1 above)
Comment (callumprentice) [inline examples/webgl_buffergeometry_selective_draw.html]: Will do - I ran it through the DoesItFitMrDoob's coding standards but missed the camelCase requirement - updating now.

File: examples/webgl_buffergeometry_selective_draw.html (hunk 2 above)
Comment (callumprentice) [inline examples/webgl_buffergeometry_selective_draw.html]: Nod - will fix.

**Comment by callumprentice** [comment]
Updated Pull Request as per suggestions - thanks and apologies for the
oversight.

https://github.com/callumprentice/three.js/commit/5662781a2ece7150c777bf5d2ffe94df740c8143

Cheers.

**Comment by makc** [comment]
You should have used bald guy from decal demo and made this into the wig

**Comment by makc** [comment]
ps @liammagee your github icon is green binoculars

**Comment by mrdoob** [comment]
Sweet! Thanks!

**Pull request merged**